### PR TITLE
REF: GeneralSettings JS > TSX

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/fugsjohecjvdh7puakrzoxmsre?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11164)

File: screen/settings/GeneralSettings.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { useContext, useEffect, useState } from 'react';
```
Comment (Overtorment) [inline screen/settings/GeneralSettings.tsx:1]: good job doing actual `git mv` so git knows its a rename, and NOT delete one file and add the other

**Review by Overtorment — APPROVED** [review]
lgtm

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**